packages/babel-core/src/transformation/block-hoist-plugin.ts

```diff
@@ -12,6 +12,7 @@ export default function loadBlockHoistPlugin(): Plugin {
     const config = loadConfig.sync({
       babelrc: false,
       configFile: false,
+      browserslistConfigFile: false,
```

Comment:
Actually we can completely avoid `loadConfig.sync`. I drafted some perf changes before but never come up with a PR: https://github.com/JLHwung/babel/commit/15da88ee201c6c09bb85f812b049f720f72b7a9a

Reply:
That would even be better! It wasn't clear to me why it was needed so I tried to work around it.

Reply:
@JLHwung  are you planning to submit your changes as a PR?

Reply:
Nope, feel free to tackle it.

Reply:
seems good as temp fix then? didn't even know we had this..